db/schema.rb

```diff
@@ -10,9 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_11_17_151338) do
+ActiveRecord::Schema.define(version: 2022_12_19_133237) do
 
-  create_table "bmap_flowcell", primary_key: "id_bmap_flowcell_tmp", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_unicode_ci", force: :cascade do |t|
+  create_table "bmap_flowcell", primary_key: "id_bmap_flowcell_tmp", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8_unicode_ci", force: :cascade do |t|
```

Comment:
revert the collate changes unless its planned as that will cause issues

Reply:
I think it should be ok. The encoding is set to the correct version. Not sure why the collate is even there. Not sure why it has removed mb3 from the collate.

Reply:
I believe rails grabs the default collate from your DMBS if it cant find the collate referenced in the schema. Quite often happens to me.

Reply:
This only happens for the warehouse! I had a similar problem with traction but adding the encoding to the database.yml removed the issue. Not sure what is different here. Maybe the Rails version.